tests/integration/conftest.py

```diff
@@ -19,11 +19,22 @@ def pytest_addoption(parser: "Parser") -> None:
     parser.addoption(
         "--gpus", action="store_true", default=False, help="run gpus related tests"
     )
+    parser.addoption(
+        "--disable-eager-execution",
+        action="store_true",
+        default=False,
+        help="Disable TF eager execution",
+    )
```

Comment:
I put this here for now since we don't have a specific sets of arguments and parser. Again, this is QA and testing and will have more discussion offline.